terraform/central-account/files/lb_private_key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
My read is that the `tls_private_key.lb-self-signed` and `tls_self_signed_cert.lb-self-signed` resources will generate a self-signed TLS cert, so it shouldn't be necessary to commit a private key to the repo. This still isn't ideal since the private key will be stored in TF state, but generating it on the fly is still very much preferable over including a default private key in the repo.

As @yi2020 mentioned in another comment, a self-signed cert will cause some healthy friction to (hopefully) push operators to use a CA-signed cert. Pending the removal of this default key, I think the per-install generation of a key is an okay default since it's at least better than defaulting to plaintext.

Reply:
Wow, I feel ridiculous. The acm.tf already did this and I forgot. File removed.